tests/.jenkins_framework.yml

```diff
@@ -8,3 +8,6 @@ FRAMEWORK:
   - flask-0.10
   - flask-0.11
   - flask-0.12
+  - elasticsearch-2
+  - elasticsearch-5
+  - elasticsearch-6
```

Comment:
WDYT about either adding another Jenkins Job for testing driver versions, or renaming`framework` to something more generic like `external_library` so you can use this for generally testing libraries in different versions against different python versions?

Reply:
yeah, definitely considering that :)